wbooks/src/app/screens/Login/index.tsx

```diff
@@ -67,7 +76,7 @@ const Login = () => {
       {errors.password && <Text style={styles.errorMessage}>{errors.password.message}</Text>}
 
       <TouchableOpacity style={styles.loginBtn} onPress={handleSubmit(onSubmit)}>
-        <Text style={styles.textLoginBtn}>Submit</Text>
+        <LoadingIndicator isLoading={responseAPILoading} />
```

Comment:
this is not a HOC

Reply:
the withLoading should be in the whole screen instead on the button's text

Reply:
Ok, I'll fix it